designs/virtualization/4.5/Scheduling/Affinity/Affinity.md

```diff
@@ -0,0 +1,63 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Affinity
+
+Introducing Affinity as it appears in-page and the Edit modal.
+
+## in-page placement
+
+![Affinity field in the Scheduling section ](img/Affinity-0-0.jpg)
+The affinity field is placed in the Workload details page, Details tab, Scheduling section, Policies columns.
+
+![Affinity field in the Scheduling section ](img/Affinity-0-1.jpg)
+The affinity field counts the number of rules applied.
+
+## Affinity rules list
+
+![Affinity all rules modal ](img/Affinity-1-0.jpg)
+On clicking on the counter Or the Edit pencil icon, the user will be presented with a modal, containing all the applied rules for this workload.
+Clicking on the 'Add affinity rule' button will replace this modal with the 'Add affinity rule' modal.
+Selecting 'Edit' from each existing rule kebab menu will replace this modal with the 'Edit affinity rule' modal
+
+A support text below the header will notify the user that:
+"Rules will ‘Preferred’ condition will stack with an ‘AND’ relation between them. 
+Rules with ‘Required condition will have an ‘OR’ relation between them."
+
+## Add/Edit affinity rule modal
+
+![Affinity Edit/Create modal ](img/Affinity-2-0.jpg)
+TThe default fields for an affinity rule are:
+- type: Node affinity
+- Condtion: Preferred
+- Weight: 1 (out of 100)
```

Comment:
is it common to know the limit is 100? Should we add "(of 100)" to the label?

Reply:
I've added it to the support text just below the field. I also added a "Lowest" and "Highest" indication to explain the scale to the user. 
Do u think it is not prominent enough? By label do you mean the field title?

Reply:
ya this is better than my idea :)
Maybe it could read 
`Value should between 1(lowest) and 100(highest)`